### PR TITLE
OSDOCS-2183: Added note about how to support a whitelist for multiple IPs or subnets

### PR DESCRIPTION
**Pull request by jonquilwilliams**
JIRA: https://issues.redhat.com/browse/OSDOCS-2183

Preview: https://deploy-preview-32609--osdocs.netlify.app/openshift-dedicated/latest/networking/routes/route-configuration.html#nw-route-specific-annotations_route-configuration

Please CP to enterprise-4.3 - 4.8

### CODE REVIEW

**Comment by jonquilwilliams** [comment]
@wgordon17  Please review this URGENT PR that you requested for OSD. Thanks.

CC: @vikram-redhat

**Comment by wgordon17** [comment]
Thanks @jonquilwilliams! 👍🏻  LGTM

File: modules/nw-route-specific-annotations.adoc [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 The Ingress Controller can set the default options for all the routes it exposes. An individual route can override some of these defaults by providing specific configurations in its annotations.
 
+[IMPORTANT]
+====
+To create a whitelist with multiple source IPs or subnets, use a space-delimited list. Any other delimiter type causes the list to be ignored without a warning or error message.
```
Comment (bmcelvee) [inline modules/nw-route-specific-annotations.adoc:12]: One question related to the conscious language project: 
could we change "whitelist" to "allowlist"?

Comment (bmcelvee) [reply modules/nw-route-specific-annotations.adoc:12]: Nevermind on this question. Looks like there are 21 instances in the file that we'd need to address.

**Comment by bmcelvee** [comment]
@quarterpin would you mind giving this PR a quick review, please? Thank you!

**Comment by quarterpin** [comment]
/lgtm 

Thanks!

**Pull request merged**